library/core/src/ops/arith.rs

```diff
@@ -78,6 +78,12 @@ pub trait Add<Rhs = Self> {
     type Output;
 
     /// Performs the `+` operation.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// assert_eq!(12 + 1, 13);
+    /// ```
```

Comment:
Should we mention what happens when add overflow on debug build and release build? IIRC it behaves differently.